packages/atomic/vitest-utils/lit-serializer.ts

```diff
@@ -0,0 +1,28 @@
+import {plugins} from '@vitest/pretty-format';
+import {SnapshotSerializer} from 'vitest';
+
+export default {
```

Comment:
Override the HTMLElement serializer: we clone the node we wanna serialize, we scrap all comment nodes from the clone, then send it back to the OG DOMElement serializer